site/src/components/StatusIndicator/StatusIndicator.tsx

```diff
@@ -15,7 +19,12 @@ export const StatusIndicator: FC<StatusIndicatorProps> = ({ color }) => {
 				height: 8,
 				width: 8,
 				borderRadius: 4,
-				backgroundColor: theme.roles[color].fill.solid,
+				backgroundColor:
+					variant === "solid" ? theme.roles[color].fill.solid : undefined,
+				border:
+					variant === "outlined"
+						? `1px solid ${theme.roles[color].outline}`
+						: undefined,
```

Comment:
similar thing here: emotion considers ternaries inside `css` objects an antipattern, and suggests using multiple objects instead (each of which becomes one css class)